src/test/java/seedu/address/testutil/DeliveryBuilder.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.testutil;
+
+import seedu.address.model.delivery.Address;
+import seedu.address.model.delivery.Delivery;
+import seedu.address.model.delivery.DeliveryName;
+import seedu.address.model.delivery.Order;
+import seedu.address.model.delivery.Phone;
+
+/**
+ * A utility class to help with building Delivery objects.
+ */
+public class DeliveryBuilder {
+
+    public static final String DEFAULT_DELIVERY_NAME = "Damith";
+    public static final String DEFAULT_PHONE = "91231231";
+    public static final String DEFAULT_ADDRESS = "Jln Parang Tritis No. 92 Orchard Road";
+    public static final String DEFAULT_ORDER = "Fried Rice 1x, Iced Kopi less sugar x1";
```

Comment:
should quantity come before item name?

Reply:
apart from this small issue, it LGTM for me!